sequence_processing_pipeline/Commands.py

```diff
@@ -33,17 +33,31 @@ def split_similar_size_bins(data_location_path, max_file_list_size_in_gb,
     current_size = max_size * 10
     fp = None
 
+    bucket_size = 0
+    max_bucket_size = 0
+
     for a, b in iter_paired_files(fastq_paths):
         r1_size = os.stat(a).st_size
+        r2_size = os.stat(b).st_size
 
         output_base = os.path.dirname(a).split('/')[-1]
         if current_size + r1_size > max_size:
+            # bucket is full.
+            if bucket_size > max_bucket_size:
+                max_bucket_size = bucket_size
+
+            # reset bucket_size to 0.
```

Comment:
This comment is not correct, right?

Reply:
read literally, it's not. However, it's the optimized version of:
bucket_size = 0
bucket_size += r1_size + r2_size

Perhaps it would be better to just say 'reset bucket_size'.